src/journal/mmap-cache.c

```diff
@@ -131,6 +131,7 @@ static void window_free(Window *w) {
         window_unlink(w);
         w->cache->n_windows--;
         free(w);
+        w = NULL;
```

Comment:
it's a local variable, setting this to NULL is not visible anywhere outside of this function, and hence pointless, since this is the last line in the function